aspnetcore/blazor/hybrid/security/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: ASP.NET Core Blazor Hybrid authentication and authorization
+author: guardrex
+description: Learn about Blazor Hybrid authentication and authorization scenarios.
+monikerRange: '>= aspnetcore-6.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 04/07/2022
+no-loc: [".NET MAUI", "Mac Catalyst", "Blazor Hybrid", Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: blazor/hybrid/security/index
+---
+# ASP.NET Core Blazor Hybrid authentication and authorization
+
+This article describes ASP.NET Core's support for the configuration and management of security in Blazor Hybrid apps.
+
+## Untrusted and unencoded content
+
+Avoid allowing an app render untrusted and unencoded content from a database or other resource, such as user-provided comments, in its rendered UI. Permitting untrusted, unencoded content to render can cause malicious code to execute.
+
+## External content rendered in an `iframe`
+
+When using an [`iframe`](https://developer.mozilla.org/docs/Web/HTML/Element/iframe) to display external content within a Blazor Hybrid page, we recommend that users leverage sandboxing features to ensure that the content is isolated from the parent page containing the app. In the following example, the [`sandbox` attribute](https://developer.mozilla.org/docs/Web/HTML/Element/iframe) is present for the `<iframe>` tag to apply [sandboxing features](https://developer.mozilla.org/docs/Web/HTML/Element/iframe) to the `foo.html` page:
+
+```html
+<iframe sandbox src="https://contoso.com/foo.html" />
+```
+
+> [!WARNING]
+> The [`sandbox` attribute](https://developer.mozilla.org/docs/Web/HTML/Element/iframe) is ***not*** supported in early browser versions. For more information, see [Can I use: `sandbox`](https://caniuse.com/?search=sandbox).
+
+## Links to external URLs
+
+By default, links to URLs outside of the app are opened using the system's browser, not loaded within the app. We do ***not*** recommend overriding the default behavior.
+
+The user might be able to indicate that they want the URL to load in the app because it's content that they trust. In that case, see the [Untrusted and unencoded content](#untrusted-and-unencoded-content) section.
+
+## Keep `BlazorWebView` (`WebView`) current in deployed apps
+
+By default, the [`BlazorWebView`](/maui/user-interface/controls/blazorwebview) control uses the currently-installed, platform-specific native `WebView`. Since the native `WebView` is periodically updated with support for new APIs and fixes for security issues, it may be necessary to ensure that an app is using a `WebView` version that meets the app's requirements.
+
+To keep the `WebView` current in deployed apps, check the `WebView` version and prompt the user to take any necessary steps to update it.
```

Comment:
I see "WebView" is `monospaced` everywhere in this document - does it need to be? To me, the monospacing implies that it refers to either some code concept or a specific WebView implementation. But most places in this document use "WebView" as a more general concept. Do you think we should switch to making it "WebView" rather than "`WebView`"?

Reply:
It was consistent with various API/UI controls, especially for consistent non-localization. I have seen it as "web view" or "Web View" with a space around the InterWebs. We can have it in text, which sounds like you prefer, as "Web View" (with a space). To avoid loc, I'll place it with our loc-avoidance marker ...

```
:::no-loc text="Web View":::
```

... but I'd like to do it everywhere else at once, so I've opened ...

 Deloc and de-fence "Web View" #25605 

... and will just focus the update for this doc at this time.

Reply:
Turns out that additional work wasn't needed, so I closed #25605. We only refer to `BlazorWebView` outside of a couple of Edge-specific `WebView2` remarks/cross-links.